tasks/ubuntu.task/preseed.erb

```diff
@@ -52,6 +53,6 @@ d-i preseed/late_command string  wget <%= log_url("preseed end") %>; \
 wget <%= file_url("os_boot") %> -O /target/usr/local/sbin/razor_postinstall.sh;  \
 printf %b '#!/bin/sh\n/usr/local/sbin/razor_postinstall.sh\nexit 0\n' > /target/etc/rc.local; \
 chmod +x /target/usr/local/sbin/razor_postinstall.sh; \
+sed -i '8i\      dhcp-identifier: mac' /target/etc/netplan/01-netcfg.yaml; \
```

Comment:
Are there any backwards compatibility concerns with adding this line?

Reply:
No. It is specific to ubuntu bionic( v 18.04) fix. 
For example: in the earlier versions xenial(16.04) the folder "/etc/netplan" doesn't exist.

Reply:
I hope this change could go in to the preseed.erb specific to ubuntu bionic.

Reply:
Please update on this.

Reply:
Apologies about the lag. Thanks for the PR. This looks good to me.